substrate/primitives/metadata-ir/src/unstable.rs

```diff
@@ -156,22 +156,24 @@ impl From<PalletConstantMetadataIR> for PalletConstantMetadata {
 
 impl From<TransactionExtensionMetadataIR> for TransactionExtensionMetadata {
 	fn from(ir: TransactionExtensionMetadataIR) -> Self {
-		TransactionExtensionMetadata {
-			identifier: ir.identifier,
-			ty: ir.ty,
-			additional_signed: ir.implicit,
-		}
+		TransactionExtensionMetadata { identifier: ir.identifier, ty: ir.ty, implicit: ir.implicit }
 	}
 }
 
 impl From<ExtrinsicMetadataIR> for ExtrinsicMetadata {
 	fn from(ir: ExtrinsicMetadataIR) -> Self {
+		// Presume all extensions work for all versions.
+		// This is true for now.
+		let transaction_extensions_by_version = ir
+			.versions
```

Comment:
That's a different version, isn't it? `ir.versions` is the transaction version (4 or 5 nowadays), but `transaction)extensions_by_version` maps the _transaction extension version_ to the list of extensions that apply. 

Thus, at the moment we'll have just one version, 0u8, which maps to the list of transaction extensions we have. I guess the `ExtrinsicMetadataIR` should be tweaked to record this information though, so that it's just mapped more 1-to-1 into this ExtrinsicMetadata?

Reply:
Yeah the `extension_version` is a different version. Here we should for now just assume `0`.